types/ember__object/evented.d.ts

```diff
@@ -1,5 +1,5 @@
 import Mixin from '@ember/object/mixin';
-import { AnyFn, EmberMethod, MethodNamesOf, MethodParams, MethodsOf } from 'ember/-private/type-utils';
+import { AnyFn, EmberMethod } from 'ember/-private/type-utils';
```

Comment:
These were unused, and just happened to get caught in this change. :sigh: Removing.